sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -30,6 +30,8 @@ AZ_NODISCARD az_storage_blobs_blob_client_options
 az_storage_blobs_blob_client_options_default(az_http_client http_client);
 typedef struct {
   struct {
+    uint8_t url_buffer[AZ_HTTP_URL_MAX_SIZE];
+    int32_t initial_url_length;
```

Comment:
If you have the uri span (next field), then you shouldn't need this field. The uri span should have the pointer, length and capacity already.